src/legacy/ui/public/courier/search_strategy/default_search_strategy.js

```diff
@@ -68,6 +68,12 @@ export const defaultSearchStrategy = {
       failedSearchRequests,
     } = await serializeAllFetchParams(allFetchParams, searchRequests, serializeFetchParams);
 
+    if (serializedFetchParams.trim() === '') {
```

Comment:
Can we add a comment to describe the meaning of this condition and how we're handling it? E.g.

```js
// If we send an empty payload we'll get back a 500 error response. If we have an empty
// payload here we can assume there are no valid requests to send, so we won't send
// the request.
```

Reply:
![giphy](https://user-images.githubusercontent.com/6239176/52867355-c4a59580-310e-11e9-986a-e0ad24a48acc.gif)